src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/contexts.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'contexts()'
+excerpt: 'xk6-browser: Browser.contexts method'
+---
+
+Allows you to access all open [BrowserContext](/javascript-api/xk6-browser/browsercontext/)s.
+
+### Returns
+
+| Type  | Description                                                                    |
+| ----- | ------------------------------------------------------------------------------ |
+| Array | Array of [BrowserContext](/javascript-api/xk6-browser/browsercontext/) objects |
+
+
+## Example
+
+<!-- eslint-skip -->
+
+```javascript
+import { chromium } from 'k6/x/browser';
+
+export default function () {
+  const browser = chromium.launch();
+  console.log(browser.contexts()); // []
+
+  const context = browser.newContext();
+  console.log(browser.contexts()); /* [
+                                        {
+                                            "base_event_emitter": {}
+                                        }
+                                      ] */
```

Comment:
```suggestion
console.log(browser.contexts().length); // 1
```

Showing people `base_event_emitter` can be confusing.

Reply:
Yep, agreed.